client/webpack.config.js

```diff
@@ -20,11 +21,10 @@ const config = {
       'react-dom': path.resolve('./node_modules/react-dom'),
     },
     modulesDirectories: [
-      "node_modules",
-      "assets/javascripts/components"
-    ],
-    root: path.resolve(".app"),
-    extensions: ['', '.js', '.jsx']
+      "node_modules",
+      "app/bundles/HomeComponent"
```

Comment:
Maybe we could specify here a whole directory instead of adding the HomeComponent explicitly? I kinda see that we will end up modifying this lien very quickly.